pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -554,7 +557,7 @@ func (t *vmRestoreTarget) Reconcile() (bool, error) {
 	if newVM.Annotations == nil {
```

Comment:
since you check this in setLastRestoreAnnotation this if is redundant

Reply:
yup